packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -348,8 +351,16 @@ export const fetchMetadata =
             throw new Error('no provider instance');
         }
 
-        const { fileName, aesKey } = entity[encryptionVersion];
-        console.debug('metadata: fatching file: ', fileName);
+        if (entity.type === 'device' && entity.status !== 'enabled') {
+            return;
+        }
```

Comment:
why this? sometimes not fetch?

Reply:
this is for TS. but basically, what would you fetch if you don't have metadata enabled? 🤔